extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -250,6 +250,7 @@ AMPHTML ad creative. Extensions or builtin tags not explicitly listed are prohib
 * [amp-img](https://www.ampproject.org/docs/reference/components/amp-img)
 * [amp-layout](https://www.ampproject.org/docs/reference/components/amp-layout)
 * [amp-lightbox](https://www.ampproject.org/docs/reference/components/amp-lightbox)
+* amp-mraid, on an experimental basis
```

Comment:
maybe add "consult @wg-ads working group if you want to use it"

Reply:
sorry, what is "@wg-ads working group"?  searching doesn't turn anything up

Reply:
https://github.com/ampproject/wg-ads

Reply:
thanks!  so I should point them to https://github.com/ampproject/wg-ads/issues/new ?

Reply:
added link